Gemfile.lock

```diff
@@ -21,6 +21,12 @@ PATH
     samson_aws_ecr (0.0.0)
       aws-sdk-ecr
 
+PATH
+  remote: plugins/datadog_tracer
+  specs:
+    samson_datadog_tracer (0.0.1)
```

Comment:
how about samson_datadog_apm ?

Reply:
The plugin just gives overlay for gem ddtrace, so I prefer to keep this name.